test/unit/template.ts

```diff
@@ -22,14 +22,27 @@ suite('Template', ({ expect, spy, itShouldBeConfigurable, itShouldHaveAlias }) =
   });
 
   describe('init()', () => {
-    it('should listen for TEMPLATE_UPDATED', () => {
+    it('should listen for TEMPLATE_UPDATED if no $sayt alias', () => {
       const on = spy();
       template.flux = <any>{ on };
       template.expose = () => null;
 
       template.init();
 
-      expect(on).to.be.calledWith(Events.TEMPLATE_UPDATED, template.updateZones);
+      expect(on).to.be.calledOnce
+        .and.calledWith(Events.TEMPLATE_UPDATED, template.updateZones);
+    });
+
+    it('should listen for AUTOCOMPLETE_TEMPLATE_UPDATED if $sayt alias found', () => {
+      const on = spy();
+      template.flux = <any>{ on };
+      template.expose = () => null;
```

Comment:
Does `expose` still need to be replaced even though it's not used in `init()`? Same with line 28.